src/main/java/com/jcabi/log/MulticolorLayout.java

```diff
@@ -137,6 +132,27 @@
      */
     private final transient Colors colors = new Colors();
 
+    /**
+     * Used to decide whether the logged text should be colored or not.
+     */
+    private final transient TextDecolorant decolorant;
+
+    /**
+     * Simple constructor.
+     */
+    public MulticolorLayout() {
+        this(new RtTextDecolorant());
+    }
+
+    /**
+     * Constructor that allos to specify the decolorant type.
+     * @param dclrnt The given decolorant.
+     */
+    public MulticolorLayout(final TextDecolorant dclrnt) {
```

Comment:
@amihaiemil we can make it visible only on the package level if it used only in tests